src/windows/views/tutorial.py

```diff
@@ -311,56 +355,63 @@ def __init__(self, win):
         # Add all possible tutorials
         self.tutorial_objects = [
             {"id": "0",
-             "x": 400,
+             "x": 0,
              "y": 0,
-             "object_id": "filesView",
+             "object_id": "dockVideoContents",
```

Comment:
```suggestion
             "object_id": "dockVideo",
```
(And in the string conditions above.) Just for clarity, since it's ultimately returning `self.win.dockVideo` now.